speakers/kristian-glass.md

```diff
@@ -0,0 +1,3 @@
+name: Kristian Glass
+---
+Etc
```

Comment:
I'll get @doismellburning to update this.  In general, I think it's up to speakers to provide sensible bios.